pynest/nest/lib/hl_api_info.py

```diff
@@ -371,7 +371,7 @@ def GetStatus(nodes, keys=None, output=''):
         raise TypeError("The first input (nodes) must be NodeCollection or a SynapseCollection with connection handles")
 
     if len(nodes) == 0:
-        return
+        return [] if output == 'json' else ()
```

Comment:
```suggestion
        return '[]' if output == 'json' else ()
```
JSON output must be a string.